docs/book/src/developer/providers/migrations/v1.4-to-v1.5.md

```diff
@@ -23,7 +23,7 @@ maintainers of providers and consumers of our Go API.
 
 ### API Changes
 
--
+- `internal/controllers/machinedeployment/mdutil.FindOldMachineSets` has dropped requiredMSs from return values.
```

Comment:
@killianmuldoon I don't think we need to track changes to internal methods, wdyt?

Reply:
You're definitely right. @PiotrLewandowski323 sorry for leading you down the wrong path there :man_facepalming: Can you revert this?